### ISSUE TITLE
The "Skip files that already exist" checkbox does not work as expected.

### ISSUE THREAD

**Issue by QuantumRevenant** [shaked6540/YoutubePlaylistDownloader#275]
If we redownload the requested playlist, it does not ignore the existing files (same name, author, duration, album and even weight). This generates the file with the format:
 * Song1.mp3
 * Song1-1.mp3
 * Song1-2.mp3
 * Song2.mp3
 * Song2-1.mp3
 * Song3.mp3

This fictitious scenario represents that I ran the download 3 times.
the first one downloaded all 3
the second downloaded up to file 2 and stopped manually
the third downloaded up to file 1 and stopped manually

I attach the configurations (in spanish)

![imagen](https://github.com/shaked6540/YoutubePlaylistDownloader/assets/94991329/29e4628f-de21-4cff-83dd-2b82c8cf973e)

**Comment by shaked6540** [comment]
Thanks for finding this, I will try to get to it when I have time

**Comment by toczekmj** [comment]
I could not reproduce that issue on my system.  
@shaked6540 have you already resolve that? 
@QuantumRevenant can you please share your operating system details?

**Comment by shaked6540** [comment]
> I could not reproduce that issue on my system. @shaked6540 have you already resolve that? @QuantumRevenant can you please share your operating system details?

Nope, I didnt have time to get to it yet

**Comment by QuantumRevenant** [comment]
@toczekmj 
I was able to reproduce the error with a different computer (multiple times)!
The program was run as administrator
The program is with the previously described configurations
The playlist was downloaded in queue.
The playlist stopped manually downloading at the second song on the first try
The playlist stopped manually at the third song on the second try
The playlist stopped manually at the second song on the third try

The results in the destination folder are as follows:
(The "SongX-1" incidents mentioned above are highlighted in red)

![image](https://github.com/shaked6540/YoutubePlaylistDownloader/assets/94991329/71b7ed86-fac6-4644-957f-35972002e813)

I attach the playlist used: https://youtube.com/playlist?list=PLsUhVbl5cUW-wLIMJgnQ_jtwUNcQBdhE_&si=ZpaWyLXLBHQKzt6T

In the following message I will attach the characteristics of this computer and those of the initial one. (It will probably be later)

**Comment by toczekmj** [comment]
Okay, thanks for the in detail informations.  I'll give it a shot and try to investigate it more in depth tomorrow.

**Comment by toczekmj** [comment]
I have found an error in above PR, please let me fix it before you take a look at it.

**Comment by toczekmj** [comment]
@QuantumRevenant you're welcome to test if it is working for you right now. You can either compile it from my fork source, or use pre-builded files https://github.com/toczekmj/YoutubePlaylistDownloader/releases/tag/Issue
(not certainly sure if I can share binaries like this, @shaked6540 correct me if I cannot so Ill remove this, and also please review #276 when you have time and will to do so.)

**Comment by QuantumRevenant** [comment]
@toczekmj Yes, it works, in English it has no problem, it skips perfectly. In Spanish there is an error "unexpected error contact the administrator", I suppose the problem is a translation issue. The song also skips, as expected, but you get this error screen 1 time for each song skipped, which is somewhat annoying. Once this fix is ​​officially published, will I generate a new issue or continue in this open issue?

![imagen](https://github.com/shaked6540/YoutubePlaylistDownloader/assets/94991329/8943f143-33a7-48f8-a9aa-7105c246c1c4)

**Comment by toczekmj** [comment]
I've checked that. It is indeed missing translation. It deservers it's own issue.
